components/edit_post_modal/edit_post_modal.tsx

```diff
@@ -249,60 +338,99 @@ class EditPostModal extends React.PureComponent {
         let message = editText;
         let newCaretPosition = this.state.caretPosition;
 
-        if (isGitHubCodeBlock(table.className)) {
-            const {formattedMessage, formattedCodeBlock} = formatGithubCodePaste(this.state.caretPosition, message, clipboardData);
+        if (typeof table !== 'boolean' && isGitHubCodeBlock(table.className)) {
```

Comment:
The `getTable` function is kind of weird, it should probably be returning `null` or `undefined` instead of false.
If possible, could we refactor that?

If not, we could just do a check for `!table` instead of checking the return type of the function to see if there is a table here.